core/examples/luigi-sample-angular/src/luigi-config/extended/helpers.js

```diff
@@ -37,18 +37,57 @@ const getProjectPlugins = projectId =>
     }
   });
 
+let allProjects = [
+  {
+    id: 'pr1',
+    name: 'Project One'
+  },
+  {
+    id: 'pr2',
+    name: 'Project Two'
+  }
+];
+
+export const addProject = newProject => {
+  allProjects.push(
+    newProject || {
+      id: 'pr' + (allProjects.length + 1),
+      name: 'Project ' + (allProjects.length + 1)
+    }
+  );
+  return allProjects[allProjects.length - 1];
+};
+
+export const removeProject = projectId => {
```

Comment:
Seems we're not using that function with parameter, so we could get rid of it and simplify the example.

Reply:
Good catch I will remove it, if needed we can always bring it back later on.